### PR TITLE
Add docker support for image server

### PR DESCRIPTION
**Pull request by sharpner**
- [x] Update Readme.md
- [x] Docker Hub Readme.md
- [x] Makefile fucks up with travis config
- [ ] Use docker-compose instead of bash script

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+COPY scripts/docker-wrapper /run-image-server
+
+# we need this for stupid ssl certificates
+# if we find a better solution for certs
+# we can run this image from busybox:ubuntu-14.04
+# and it will be a looot smaller
+RUN apt-get update
+RUN apt-get install -y ca-certificates 
+
+CMD /bin/sh run-image-server
```
Comment (dazorni) [inline Dockerfile]: No new line at end file

**Comment by sharpner** [comment]
fixes #11

**Comment by sharpner** [comment]
fixes #9

**Comment by sharpner** [comment]
this needs much rework. It's not relevant for now.

**Comment by dazorni** [comment]
Ohhh :cry:

**Comment by sharpner** [comment]
I couldn't find a proper service that will autobuild those containers. If you'd like, I'm open to PRs  (and finish it up) if you can find a solution for this ;)

**Comment by dazorni** [comment]
I will take a look, thanks :+1:

**Pull request closed without merging**